frontend/src/Providers.tsx

```diff
@@ -25,45 +24,19 @@ const getOidcConfig = async () => {
 };
 
 const onSigninCallback = (): void => {
-  window.history.replaceState({}, document.title, window.location.pathname);
+  // window.history.replaceState({}, document.title, window.location.pathname);
```

Comment:
Is this relevant?

```suggestion
```

Reply:
I think it is.
It is done this way in the function description of the onSigninCallback of 'react-oidc-context'.
https://github.com/authts/react-oidc-context/blob/f175dcba6ab09871b027d6a2f2224a17712b67c5/src/AuthProvider.tsx#L20-L30
The autorization page redirects to our page with some url parameters set, and this clears the parameters, if the login was successful.

Reply:
Was a mistake that I left the commented out code there.